### PR TITLE
PIXIE

### PR DESCRIPTION
**Pull request by orlof**

### CODE REVIEW

**Comment by paultech** [comment]
Should we be listing the interrupts for the LEM also? Or assumed they will read the LEM spec?

**Comment by orlof** [comment]
Perhaps the startup company (Agnus Micro Devices), that developed the breadboard for PIXIE chipset, is brave enough to include that part of the industry giant Nya Elektriska's documentation.

File: Displays/PIXIE.txt [hunk 1]
```diff
@@ -0,0 +1,117 @@
+PIXIE
+
+
+DCPU-16 Hardware Info:
+    Name: PIXIE - Compatible Video Generator
+    ID: 0x774df615, version: 0x1802
+    Manufacturer: 0x00000000 (Manufacturer not yet found)
```
Comment (andyleap) [inline Displays/PIXIE.txt]: I'd recommend going ahead and making up a value for Agnus Micro Devices or whatever and put that in here

File: Displays/PIXIE.txt [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    word addresses 20 through 787. Each of these addresses accesses
+    one word (16 pixels) of a single bitplane. Image with 16 colors
+    has four bitplanes like this.
+
+Hardware ID
+    PIXIE hardware id is 0x774df615, while LEM1802 hardware id is
+    0x734df615. To guarantee full compatibility with all LEM1802
+    software, PIXIE has a dip switch located in the back side.
+
+    Dip swicth has two positions that can be used to change the
```
Comment (andyleap) [inline Displays/PIXIE.txt]: Dip switch*

Comment (andyleap) [reply Displays/PIXIE.txt]: Also, is the "graphics mode" locked out in position 0 (LEM1802), or is it still there?

File: Displays/PIXIE.txt [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        e.g. if bitplane 0 is clear, but bitplanes 1-3 are set,
+             the palette color would be 0b1110 -> 0xE -> 14
+
+    Gaphics are stored to bitplane in a linear fashion; each word of data
+    on a line is located at an address that is one greater than the word on
+    its left. i.e.  Each line is a "plus one" continuation of the previous
+    line.
+                   _______________________________________
+                  |    |    |    |    |    |    |    |    |
+                  | 20 | 21 | 22 | 23 | 24 | 25 | 26 | 27 |
```
Comment (andyleap) [inline Displays/PIXIE.txt]: Is there some reason this bitplane is starting at 20, whereas the table above lists 0 as the base offset for the first bitplane?  (If you are showing this as "here's how it'd look if your MEM_MAP_SCREEN was 20", maybe make more explicit?

File: Displays/PIXIE.txt [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+          0: Text mode (LEM1802 standard)
+          1: Graphics mode, 2 colors
+          2: Graphics mode, 4 colors
+          3: Graphics mode, 8 colors
+          4: Graphics mode, 16 colors
+
+       Graphics mode will use the same video_ram starting address as text mode,
+       but video_ram size and layout are different. Graphics mode will also use
+       the same palette as text mode, but only the colors 0-n (depending on the
+       color depth) are used. Fontram is not relevant in graphics mode.
```
Comment (andyleap) [inline Displays/PIXIE.txt]: font ram and video ram are referred to via "font ram" and "video ram", could we drop the underscores, and add a space to Fontram?

File: Displays/PIXIE.txt [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+    ID: 0x774df615, version: 0x1802
+    Manufacturer: 0x00000000 (Manufacturer not yet found)
+
+
+Description:
+    The PIXIE is a 128x96 pixel color display fully compatible with the
+    famous Nya Elektriska's LEM1802. PIXIE is a perfect way to upgrade
+    your DCPU and start enjoying the latest games and graphics.
+
+    PIXIE has the same interrupt behaviour as LEM1802, but it also supports
```
Comment (andyleap) [inline Displays/PIXIE.txt]: behaviour == behavior... second is american, first is everywhere else.  Let's make the spec consistent!  If "AMD"(lol) is going to be "british-y" or whatever, change all the variants, like color => colour  (Note, not saying consistent with the rest of the specs, but if a british company was releasing specs, all their specs would likely be consistent with each other)  If you aren't going for that "british-y" kind of thing, just switch it to behavior.

**Review by andyleap — COMMENTED** [review]
Just a few minor tweaks.

**Pull request merged**